tls/test/test_hello_message.py

```diff
@@ -139,6 +139,18 @@ class TestClientHello(object):
         b'\x00\x04'
     ) + truncated_hmac_ext_packet
 
+    cert_status_request_ext = (
```

Comment:
From the rfc:
```
  A zero-length "responder_id_list" sequence has the special meaning that
  the responders are implicitly known to the server, e.g., by prior arrangement.
```
and:
```
   A zero-length "request_extensions" value means that there are no
   extensions (as opposed to a zero-length ASN.1 SEQUENCE, which is not
   valid for the "Extensions" type).
```

I need to come up with some test bytes that's not zero-length request_extensions and responder_id_list.